### ISSUE TITLE
Optional verbose, or no verbose

### ISSUE THREAD

**Issue by leonnleite** [aszonecombr/search-hacking#5]
print a log is an optional feature.

I don't like see the flood in my terminal (or browser) in any command.

Also, should have only a responsibility... when I print log and search a hacking, I do two different things

**Comment by abdala** [comment]
I agree with you. We'll add PSR 3 LoggerInterface, like we are doing in aszone/vulnerabilities.

**Comment by leonnleite** [comment]
@lenonleite  could u commit in aszone/vulnerabilities?

I'm waiting for this
